### PR TITLE
support android 10 (API 29)

### PR DESCRIPTION
**Pull request by eliadAfeka**
### Description

added as a fix for https://github.com/apache/cordova-plugin-file/issues/408

### Checklist

- [ ] I've run the tests to see all new and existing tests pass
- [ ] I added automated test coverage as appropriate for this change
- [ ] Commit is prefixed with `(platform)` if this change only applies to one platform (e.g. `(android)`)
- [ ] If this Pull Request resolves an issue, I linked to the issue in the text above (and used the correct [keyword to close issues using keywords](https://help.github.com/articles/closing-issues-using-keywords/))
- [ ] I've updated the documentation if necessary

### CODE REVIEW

**Comment by breautek** [comment]
Blocked by https://github.com/apache/cordova-plugin-file/pull/411

File: plugin.xml [hunk 1]
```diff
@@ -132,7 +132,9 @@ to config.xml in order for the application to find previously stored files.
             </feature>
             <allow-navigation href="cdvfile:*" />
         </config-file>
-
+		<edit-config file="app/src/main/AndroidManifest.xml" mode="merge" target="/manifest/application">
+					<application android:requestLegacyExternalStorage="true" />
```
Comment (breautek) [inline plugin.xml:136]: Can you fix the tabbing here? Thanks

```suggestion
		<edit-config file="app/src/main/AndroidManifest.xml" mode="merge" target="/manifest/application">
		    <application android:requestLegacyExternalStorage="true" />
```

**Comment by breautek** [comment]
@eliadAfeka Looks like you just reverted this PR via https://github.com/apache/cordova-plugin-file/pull/410/commits/28aa7121bc1968e96e21141e5c2a4c4a53813ebf, so that this no has no changes

**Comment by jitendracodehacker** [comment]
Hi , When will you have support to API29. this has stopped our release as file-plugin fails.

**Comment by felipecaparelli** [comment]
I'm just wondering, it this plugin.xml change the only thing needed to be compliant with Android 10?

**Comment by breautek** [comment]
> I'm just wondering, it this plugin.xml change the only thing needed to be compliant with Android 10?

From what I've seen, yes.

**Comment by eliadAfeka** [comment]
hi,

sorry for the rush, but is everything ok with this PR? if yes, when you are going to upload it?
this plugin is critical in my projects.

thanks.

**Comment by smartyw** [comment]
I believe this PR, which was supposed to fix this issue was merged so I built against head just now but still get the same error INVALID_MODIFICATION_ERR  when running on Android 10. Is this expected? Do I need to wait for an official release of the plugin? There's some urgency to get applications using target SDK 29 by November. Thanks.

**Comment by EinfachHans** [comment]
@breautek Any chance we can get a new released Version here?

**Pull request closed without merging**